torchserve/datadog_checks/torchserve/openmetrics_check.py

```diff
@@ -0,0 +1,36 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.base import OpenMetricsBaseCheckV2
+from datadog_checks.base.checks.openmetrics.v2.scraper import OpenMetricsScraper
+from datadog_checks.torchserve.config_models import ConfigMixin
+from datadog_checks.torchserve.metrics import OPENMETRICS_METRIC_MAP
+
+
+class TorchServeOpenMetricsScraper(OpenMetricsScraper):
+    SERVICE_CHECK_HEALTH = "health"
+
+
+class TorchserveOpenMetricsCheck(OpenMetricsBaseCheckV2, ConfigMixin):
+    __NAMESPACE__ = 'torchserve.openmetrics'
+    DEFAULT_METRIC_LIMIT = 0
+
+    def create_scraper(self, config):
+        # We prefix all the metrics with their endpoint. If we use the default scraper, we will emit the service check
```

Comment:
Move this comment to line 12. Or we could also keep this comment here and change line 23 to something like

```
scraper = OpenMetricsScraper(self, self.get_config_with_default(config))
scraper.SERVICE_CHECK_HEALTH = "health"
return scraper
```

Reply:
Helpful comment 👍

Reply:
good idea